docs/_data/spacing.json

```diff
@@ -370,6 +382,10 @@
       "term": "ln",
       "define": "negative left"
     },
+    {
+      "term": "ln",
```

Comment:
This should be `in` right? I wonder if we even need negative inset. That seems rare?

Reply:
> This should be `in` right?

Yep, you caught a typo. I'll push a fix up.

> I wonder if we even need negative inset. That seems rare?

Yeah, it kinda goes against the concept of "inset". I included it just for consistencies sake, but it's probably solidly in YAGNI territory. I'll get rid of it.